frontend/src/UserList.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import {IPlayer} from './types';
+import styled from 'styled-components';
+import EmojiButton from './EmojiButton';
+import {WebsocketContext} from './WebsocketProvider';
+
+const Container = styled.div`
+  width: 80%;
+  border: 1px solid #eee;
+  border-radius: 4px;
+  padding: 1em;
+`;
+
+const Row = styled.div`
+  display: table-row;
+  border-collapse: separate;
+`;
+
+const Cell = styled.div`
+  display: table-cell;
+  border-style: solid;
+  border-color: transparent;
+  border-width: 0.2em 1em;
+`;
+
+const FullWidthCell = styled.div`
+  text-align: center;
+`;
+
+const Heading = styled(FullWidthCell)`
+  font-size: 1.2em;
+  padding: 0.5em;
+  font-weight: 200;
+`;
+
+const Empty = styled(FullWidthCell)`
+  font-style: italic;
+  color: #555;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  flex-direciton: row;
+  justify-content: flex-end;
+  width: 10em;
+  & > * {
+    margin: 0px 0.3em;
+  }
+`;
+
+type RowProps = {
+  player: IPlayer;
+  actions: JSX.Element;
+};
+const UserRow = (props: RowProps) => (
+  <Row key={props.player.name}>
+    <Cell style={{width: '50%'}}>{props.player.name}</Cell>
+    <Cell style={{width: '50%'}}>Rank {props.player.level}</Cell>
```

Comment:
What do you think about minimizing the space dedicated to the level (and skewing it more towards the player name).

Similarly, we can reduce the size of the overall table and increase readability that way -- the maximum player name length is 32 chars

Reply:
This is less of an issue if the overall box is narrower as suggested below

Reply:
Changed "Rank" to be minimum width, so players column will eat up all available space